server/src/main/java/org/elasticsearch/search/aggregations/timeseries/TimeSeriesIndexSearcher.java

```diff
@@ -168,6 +170,11 @@ int nextDoc() throws IOException {
             return docId;
         }
 
+        BytesRef getTsid() throws IOException {
+            scratch.copyBytes(tsids.lookupOrd(tsids.ordValue()));
```

Comment:
I thought you said no production code changes 😄

Reply:
Hah, this was to help debugging but I think it's also safer so I left it in